### DISCUSSION TITLE
Performance Issue | Finding Record taking more than 1.2 Seconds with Simple Where Condition

### DISCUSSION THREAD

**Discussion by myalgomate** [litedb-org/LiteDB#2529] [General]
We are finding this record with Linq in C# that is also taking the same time Around >1.2 Sec
so we tried regenerating the where condition with DB viewer with that also we are getting the same delay.

We are using Version [v5.0.17] currently.

SELECT $ FROM Custom_Instrument where InstrumentToken ="867726";

Uploading Sample file 
https://filebin.net/ncrc1l4wseq3s7q8#

Total Records: 74,000

How can we decrease the time ..?

![image](https://github.com/user-attachments/assets/4d59f168-b5f9-4338-adb6-eb11c2f73232)

**Comment by usercode** [comment]
I guess you forget to use an index for this column.

Look here: 
https://www.litedb.org/docs/indexes/
